src/freenas/etc/ix.rc.d/ix-update

```diff
@@ -104,7 +104,14 @@ db_update()
 			rm ${HBA_FIRMWARE_UPDATE}
 		fi
 	fi
-			
+
+	if [ -f $NEED_UPDATE_SENTINEL ]; then
```

Comment:
Isnt this only actually used for config upload? For actual upgrades it needs to be in build repo, right?